packages/scripts/templates/feature/${featureName.dashCase}.main.env.tsx.tmpl

```diff
@@ -0,0 +1,15 @@
+import ${featureName.pascalCase} from './${featureName.dashCase}.feature';
+
+// import { Environment } from '@wixc3/engine-core';
+// const mainEnv = new Environment('main', 'window', 'single');
+
+${featureName.pascalCase}.setup(mainEnv, ({ run, ${featureName.camelCase}Options }, {}) => {
+    // eslint-disable-next-line no-console
+    console.log(${featureName.camelCase}Options);
```

Comment:
this is weird to create a code with eslint-disable-next-line no-console. What if I am using jest? tslint? or any other?
maybe consider removing the console.log